### ISSUE TITLE
misleading error at 'from ssv' raising 'can not convert string to integer'

### ISSUE THREAD

**Issue by avkonst** [nushell/nushell#6830]
### Describe the bug

The following pipeline in the script reports the error:
```
^ps -ef | ^grep socat | from ssv --noheaders | rename uuid pid ppid dummy command
```
![image](https://user-images.githubusercontent.com/3971413/197077658-55cfbd17-9dd3-4a3f-9a1b-a61739537365.png)

running it in the terminal is just fine and works

### How to reproduce

run some socat processes and run the following in the script:
```
^ps -ef | ^grep socat | from ssv --noheaders | rename uuid pid ppid dummy command
```

### Expected behavior

no error raise on `from ssv`

### Screenshots

provided above

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.69.1                                          |
| branch             |                                                 |
| commit_hash        | 8a5abc7afc28a36908a32f1852127e2a5170b461        |
| build_os           | linux-x86_64                                    |
| build_target       | x86_64-unknown-linux-musl                       |
| rust_version       | rustc 1.64.0 (a55dd71d5 2022-09-19)             |
| rust_channel       | stable-x86_64-unknown-linux-gnu                 |
| cargo_version      | cargo 1.64.0 (387270bc7 2022-09-16)             |
| pkg_version        | 0.69.1                                          |
| build_time         | 2022-09-28 02:54:35 +00:00                      |
| build_rust_channel | release                                         |
| features           | database, dataframe, default, trash, which, zip |
| installed_plugins  |                                                 |

**Comment by avkonst** [comment]
OK. It turned out the error is misleading. I do kill on the subsequent pipeline and this is where it was really unhappy:
```
$processes | each { |$it| echo { stoping: $cmd }; kill $it.pid }
```
the following fixed it:
```
$processes | each { |$it| echo { stoping: $cmd }; kill ( $it.pid | into int ) }
```

**Comment by avkonst** [comment]
PS: I can not use built in ps due to https://github.com/nushell/nushell/issues/6829

**Comment by sholderbach** [comment]
I struggle to reproduce the error with `0.73`. ( I get a result for the `grep name` as well and nothing when disabling the `ps` `-f` flag) 

I think some of the ingest stage of the from commands has been improved since. (Maybe the fix to #6829 already practically voided the problem for ou @avkonst)

**Comment by sholderbach** [comment]
Closing for now feel free to reopen if the problem persists.
